Rakefile

```diff
@@ -44,11 +45,13 @@ task :syntax do
   end
 end
 
-task :rubocop do
-  require 'rubocop'
-  cli = RuboCop::CLI.new
-  exit_code = cli.run(%w(--display-cop-names --format simple))
-  raise "RuboCop detected offenses" if exit_code != 0
+RuboCop::RakeTask.new(:rubocop) do |task|
```

Comment:
Its cleaner!

Reply:
This plus the require above also make tasks from Rubocop show up in `bundle exec rake -T`